Makefile

```diff
@@ -31,10 +31,10 @@ help:  ## Display this help
 
 .PHONY: gazelle
 gazelle: ## Run Bazel Gazelle
-	(which bazel && bazel run //:gazelle) || true
+	(which bazel && ./hack/update-bazel.sh) || true
```

Comment:
Not sure if we want to keep this script but as long as we have it we should IMHO use it :)

Reply:
I assumed the `/hack` folder script usually target the CI environment, I'm ok using it, although I haven't seen other projects doing this yet.

Reply:
test-infra is my go-to for best-practice here.  They seem to be using it for all their development-time scripts.  Maybe we should call it `dev` rather than `hack` but I think it's a good idea to try to have a canonical location for each task - splitting between the makefile and the scripts leads to problems IMO.